handpose/src/index.ts

```diff
@@ -23,29 +23,25 @@ import {MESH_ANNOTATIONS} from './keypoints';
 import {Coords3D, HandPipeline, Prediction} from './pipeline';
 
 // Load the bounding box detector model.
-async function loadHandDetectorModel() {
-  const HANDDETECT_MODEL_PATH =
-      'https://tfhub.dev/mediapipe/tfjs-model/handdetector/1/default/1';
-  return tfconv.loadGraphModel(HANDDETECT_MODEL_PATH, {fromTFHub: true});
+async function loadHandDetectorModel(handdetectModelURL:string) {
+  return tfconv.loadGraphModel(handdetectModelURL, {fromTFHub: true});
```

Comment:
If you pass in a custom URL, we probably shouldn't pass in {fromTFHub: true} here.

Reply:
Thank you. 

From the `loadGraphModel` document:
const modelUrl = 'https://storage.googleapis.com/tfjs-models/savedmodel/mobilenet_v2_1.0_224/model.json';
const model = await tf.loadGraphModel(modelUrl);

const modelUrl =  'https://tfhub.dev/google/imagenet/mobilenet_v2_140_224/classification/2';
const model = await tf.loadGraphModel(modelUrl, {fromTFHub: true});

When passing `loadOptions` =  {fromTFHub: true}, it just means the modelURL ends with a folder ?

So I suppose the easiest way to clear away confusion is  refining the comments of `load` function?  like:
- `handdetectModelURL`  Specify  tfhub mirror site URL of handdetect model. Defaults to tfhub.dev.

By the way, other tfjs-models like `speech-commands`, the custom URL is similar to storage.google.